docs/building.md

```diff
@@ -45,7 +45,7 @@ Here are few commands that you will likely use the most:
     - `build.sh --restore`: to install the required .NET SDK, .NET tools and the toolset. This is equivalent to running `./restore.sh`.
     - `build.sh --build`: to build the solution<sup>1</sup>.
     - `build.sh --test`: to run all unit tests in the solution<sup>1</sup>.
-    - `build.sh --vs <keywords>`: to generate a "filtered" solution and save it as `SDK.sln`. It also performs the "restore" operation. For example: `./build.sh --vs Http,Fakes`.<br />
+    - `build.sh --vs <keywords>`: to generate a "filtered" solution and save it as `SDK.sln`. It also performs the "restore" operation. Keywords can be any part of the name or path of project files you want to include. For example: `./build.sh --vs Http,Fakes,AspNetCore`.<br />
```

Comment:
is it intended that it's --vs here and -vs below

Reply:
Good catch, I don't think this is intentional.

Reply:
Our bash shell scripts accept a single dash (e.g., `-vs`) for the compat with the ps1 scripts.
https://github.com/dotnet/extensions/blob/c8e34c57199a2e5e3864649f3bd16258b88ff67e/eng/build.sh#L34-L41